charts/jenkins/README.md

```diff
@@ -57,13 +57,36 @@ To see all configurable options with detailed comments, visit the chart's [value
 ```console
 # Helm 3
 $ helm show values jenkins/jenkins
-
-# Helm 2
-$ helm inspect values jenkins/jenkins
 ```
 
 For a summary of all configurable options, see [VALUES_SUMMARY.md](./VALUES_SUMMARY.md)
 
+### Configure Security Realm and Authorization Strategy
+
+This chart configured a `securityRealm` and `authorizationStrategy` as shown below:
+
+```yaml
+controller:
+  JCasC:
+    securityRealm: |-
+      local:
```

Comment:
IIUC top-level entries in JCasC are `String`-valued map entries in this chart? Weird.